src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/DateTimeOffsetTestExtensions.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+
+public static class DateTimeOffsetTestExtensions
+{
+    /// <summary>
+    /// Strip the nanosecond component from a DateTimeOffset. Helpful for testing in scenarios where precision can
+    /// change throughout a DateTimeOffset's lifecycle.
+    ///
+    /// An example would be when comparing DateTimeOffsets when one has been retrieved from PostgreSQL, where the
+    /// precision is less than that of C# itself.
+    /// 
+    /// </summary>
+    /// <param name="offset"></param>
+    /// <returns></returns>
+    public static DateTimeOffset TruncateNanoseconds(this DateTimeOffset offset)
```

Comment:
This doesn't need to be a test extension - it can just be in Common as we could easily re-use this in non-test code as well

Reply:
I've stuck an Assert.NotNull() in there now to ensure you're passing in a non-null DateTimeOffset during testing (which has actually caught a test setup that was incorrectly assuming a non-null DateTimeOffset was present where it was in fact not, so I'd be keen to keep it in Test for now.  Happy to rejig and move it in the future if we ever have the need though?

Reply:
Actually, why not eh?  I'll revert it to throw ArgumentException and move it into a production project